internal/common/common.go

```diff
@@ -432,6 +435,22 @@ func GetInventoryInterfaces(inventory string) (string, error) {
 	return inventory[interfacesLocation : interfacesLocation+endLocation], nil
 }
 
+// blocking host removal while cluster is progressing
+func CanUnbindhost(c *Cluster) (err error) {
+	clusterStatus := swag.StringValue(c.Status)
+	NotAllowedStatuses := []string{
+		models.ClusterStatusFinalizing,
+		models.ClusterStatusInstalling,
+		models.ClusterStatusInstallingPendingUserAction,
```

Comment:
I can think about scenario were i would actually like to remove node in ClusterStatusInstallingPendingUserAction.
Lets say worker is stuck in it and i have more workers wouldn't i want to continue installation and not handle stuck worker? @avishayt ?

Reply:
I agree, this seems like a likely use case for wanting to remove the node.

Reply:
It leave some room for people to shoot themselves in the foot, but I'd be OK with that compromise.

Reply:
I think we should advise the user to cancel the installation and then remove nodes.

so when, in `ClusterStatusInstallingPendingUserAction` we are still progressing the installation thus we don't want user to remove nodes.